api/block/schema.py

```diff
@@ -1,16 +1,43 @@
 import graphene
-
+from graphql import GraphQLError
 from graphene_sqlalchemy import SQLAlchemyObjectType
 from api.block.models import Block as BlockModel
 from api.room.schema import Room
+from api.office.models import Office
 from helpers.room_filter.room_filter import room_join_location
+from utilities.utility import validate_empty_fields
+from helpers.auth.authentication import Auth
 
 
 class Block(SQLAlchemyObjectType):
     class Meta:
         model = BlockModel
 
 
+class CreateBlock(graphene.Mutation):
+    class Arguments:
+        name = graphene.String(required=True)
+        office_id = graphene.Int(required=True)
+    block = graphene.Field(Block)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, **kwargs):
+        validate_empty_fields(**kwargs)
+        block_name = BlockModel.query.filter_by(name=kwargs['name'].title()).all() # noqa
+        if len(block_name) > 0:
+            raise GraphQLError("Block aleady exists")
+        get_office = Office.query.filter_by(id=kwargs['office_id']).first()
+        if get_office is None:
```

Comment:
simply use implicit boleannes here, like if get_office: ...

Reply:
@shemogumbe  cool on it

Reply:
@shemogumbe  I have impemented the changes